locales/de.json

```diff
@@ -959,5 +959,6 @@
 	"5": "Windy",
 	"6": "Snow",
 	"7": "Fog"
-  }
+  },
+  "boosted": "verstärkt"
```

Comment:
Instead of leaving this out, what do you think of using a 'Misc' subsection for random words that we need translations for? I'm just worried this 'boosted' section will get lost if we process this file. Do you feel like this is a concern?

Reply:
Fine with me. I was kinda thinking that when I made it anyway 😄